fastlane/lib/fastlane/actions/docs/upload_to_play_store.md

```diff
@@ -83,6 +83,26 @@ Expansion files (obbs) found under the same directory as your APK will also be u
 - they are identified as type 'main' or 'patch' (by containing 'main' or 'patch' in their file name)
 - you have at most one of each type
 
+### Expansion files (`.obb`)
```

Comment:
That's wrong here, the lines above should also be covered by this headline.

Should I go in and make the changes myself?

Reply:
Updated